serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc

```diff
@@ -168,35 +168,27 @@ Kubernetes service discovery is disabled::
 ----
 ====
 
-[IMPORTANT]
-====
+[[ref-enabling-kubernetes-service-discovery]]
+== Enabling Kubernetes service discovery
+
 When using the Kubernetes service discovery feature, you need to balance if your application can afford the delayed startup time.
 
 If the URI pattern is not found in the application properties, then discovery is not triggered. However, the scanning is performed anyway. Therefore, a short time is required to go through the startup scan.
 
-You can disable the Kubernetes service discovery by removing the `kogito-addons-quarkus-kubernetes` library from the application's dependencies as shown in the following Maven exclusions:
+You can enable the Kubernetes service discovery by adding a service discovery implementation to the application's dependencies as shown in the following Maven dependencies:
```

Comment:
If I get it right, Service Discovery is deactivated by default, which means we need to specific the non backward into the release notes

Reply:
Do you mean adding it to the release notes in this PR or the PR we usually have at the end of the release cycle?

Reply:
To be added to the release notes once we create it for 1.39